pkgs/top-level/all-packages.nix

```diff
@@ -17766,6 +17766,7 @@ with pkgs;
     inherit (darwin.apple_sdk.frameworks) OpenGL;
     inherit (darwin.apple_sdk.libs) Xplugin;
   };
+  mesa-opencl = mesa.override { enableOpenCL = true; };
```

Comment:
Considering there's already a `mesa_glu`, it should probably be named `mesa_opencl` for consistency.